.ci_support/linux_64_.yaml

```diff
@@ -6,3 +6,5 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
```

Comment:
Just a note, I wouldn't change this file by hand. Typically we let the bots modify these files on re-render.

Reply:
I've only been modifying meta.yml and letting smithy do the rest.